### PR TITLE
API-42925-tracked_item-serv-standardization 

### PR DESCRIPTION
**Pull request by stiehlrod**
## Summary

- Removes extra methods from definitions file and 
- removes corresponding tests.

## Related issue(s)

- [API-42925](https://jira.devops.va.gov/browse/API-42925)

## Testing done

- [x] Tested V2 claims show
- [x] Tested V2 claims show in staging and search DD for wsdl
- [x] Tested V1 claims show Flipper.enable :claims_status_v1_bgs_enabled
- [ ] Tested V1 claims show in staging and search DD for wsdl

## What areas of the site does it impact?
	modified:   modules/claims_api/app/clients/claims_api/bgs_client/definitions.rb
	modified:   modules/claims_api/spec/lib/claims_api/find_definition_spec.rb

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [X]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

### CODE REVIEW

**Review by tycol7 — APPROVED** [review]

File: modules/claims_api/app/clients/claims_api/bgs_client/definitions.rb [hunk 1]
```diff
@@ -200,7 +200,6 @@ module StandardDataWebService
       ##
       # TrackedItemService
       #
-      # Adding 'Bean' to the end to differentiate from the service
       module TrackedItemServiceBean
```
Comment (tycol7) [inline modules/claims_api/app/clients/claims_api/bgs_client/definitions.rb:203]: I thought we were removing Bean from the module names? (e.g., https://github.com/department-of-veterans-affairs/vets-api/pull/19791/files)

Comment (tycol7) [reply modules/claims_api/app/clients/claims_api/bgs_client/definitions.rb:203]: Never mind, I see the revert in #19934.

File: modules/claims_api/spec/lib/claims_api/find_definition_spec.rb [hunk 2]
```diff
@@ -151,18 +136,6 @@
...
-        let(:endpoint) { 'TrackedItemService/TrackedItemService' }
-
-        it 'response with the correct namespace' do
-          result = subject.for_service(endpoint)
-          parsed_result = JSON.parse(result.to_json)
-          expect(parsed_result['bean']['path']).to eq 'TrackedItemService'
-          expect(parsed_result['path']).to eq 'TrackedItemService'
-          expect(parsed_result['bean']['namespaces']['target']).to eq 'http://services.mapd.benefits.vba.va.gov/'
-        end
-      end
```
Comment (tycol7) [inline modules/claims_api/spec/lib/claims_api/find_definition_spec.rb:164]: Isnʼt this test still good?

Comment (FonzMP) [reply modules/claims_api/spec/lib/claims_api/find_definition_spec.rb:164]: should be.

**Pull request closed without merging**